core/execution/market.go

```diff
@@ -1964,19 +1948,14 @@ func (m *Market) resolveClosedOutParties(ctx context.Context, distressedMarginEv
 		size = uint64(-networkPos)
 	}
 
-	ref := "LS"
-	if orderID != nil {
-		ref = fmt.Sprintf("LS-%s", *orderID)
-	}
-
 	no := types.Order{
 		MarketID:    m.GetID(),
 		Remaining:   size,
 		Status:      types.OrderStatusActive,
 		Party:       types.NetworkParty, // network is not a party as such
 		Side:        types.SideSell,     // assume sell, price is zero in that case anyway
 		CreatedAt:   now.UnixNano(),
-		Reference:   ref,                       // liquidity sourcing, reference the order which caused the problem
+		Reference:   "LS",                      // liquidity sourcing, reference the order which caused the problem
```

Comment:
[not important] why not keep the convention of ref = fmt.Sprintf("LS-%s", *orderID) just do it yourself here with m.idgen.NextID()?

Reply:
Can do, just didn't think it was adding any value (that order will have its ID like any other one), some random reference didn't seem like it'd help with anything, but I can put it back in.

Reply:
I think it didn't have sense one we were MTMing after each trade, then it'd would get the ID of the order that resulted in the price move that brought about the closeout, but after moving to time-based MTM it lost it's meaning.